coderdojochi/settings.py

```diff
@@ -25,7 +26,7 @@
 SECRET_KEY = 'e^u3u$pukt$s=6#&9oi9&jj5ow6563fuka%y9t7i*2laalk^l$'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = str_to_bool(os.environ.get('DEBUG')) or False
+DEBUG = str_to_bool(os.environ.get('DEBUG')) and 'test' not in sys.argv or False
```

Comment:
@karbassi when I was writing tests I was trying to check response url, but the redirect was being intercepted by the redirect debug page and my response url was wrong.  Let me know if you need more explanation